### ISSUE TITLE
Accessibility: jupyter.org/install

### ISSUE THREAD

**Issue by Raymond-techy** [jupyterhub/outreachy#64]
### What page is this for?

https://docs.jupyter.org/en/latest/install.html

### WAVE accessibility report

1 X Missing alternative text
Missing alternative text
1 X Empty form label
24 X Very low contrast
![Jupyter](https://user-images.githubusercontent.com/81036251/196047811-db519f5a-17ed-4f83-bffe-d3795306c3e4.PNG)

### The HTML element to be changed

```html
<img src="_static/jupyter.svg" class="logo__image only-light" alt="Logo image">
<img src="_static/jupyter.svg" class="logo__image only-dark" alt="Logo image">

<label for="toctree-checkbox-1">
<i class="fas fa-chevron-down"></i>
</label>

<a class="reference internal" href="install/kernels.html" style="color: rgb(69, 157, 185); background-color: rgb(255, 255, 255);">
Installing Kernels
</a>

<a class="current reference internal" href="#">
Install and Use
</a>
<a class="reference internal" href="install/kernels.html#how-do-i-install-python-2-and-python-3">
How do I install Python 2 and Python 3?
</a>
<a class="reference internal" href="install/kernels.html#how-do-i-install-other-languages-like-r-or-julia">
How do I install other languages like R or Julia?
</a>

<p class="prev-next-title">
Try Jupyter
</p>

<p class="prev-next-title">
Installing the classic Jupyter Notebook interface
</p>

<a href="http://sphinx-doc.org/">
Sphinx
</a>
```

### Your proposed HTML (or CSS), after change

```html
//add a comprehensive alternative text
<img src="_static/jupyter.svg" class="logo__image only-light" alt="jupyter Logo image">
<img src="_static/jupyter.svg" class="logo__image only-dark" alt="jupyter Logo image">

//remove the label tag
<i class="fas fa-chevron-down"></i>

//improving text contrast
<a class="reference internal" href="install/kernels.html" style="mix-blend-mode: difference;
color: rgb(69, 157, 185); background-color: rgb(255, 255, 255);">
Installing Kernels
</a>

<a class="current reference internal" href="#" style="mix-blend-mode: difference;">
Install and Use
</a>
<a style="mix-blend-mode: difference;" class="reference internal" href="install/kernels.html#how-do-i-install-python-2-and-python-3">
How do I install Python 2 and Python 3?
</a>
<a style="mix-blend-mode: difference;" class="reference internal" href="install/kernels.html#how-do-i-install-other-languages-like-r-or-julia">
How do I install other languages like R or Julia?
</a>

<p class="prev-next-title" style="mix-blend-mode: difference;">
Try Jupyter
</p>

<p class="prev-next-title" style="mix-blend-mode: difference;">
Installing the classic Jupyter Notebook interface
</p>

<a href="http://sphinx-doc.org/" style="mix-blend-mode: difference;">
Sphinx
</a>
```

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
